frontend/src/entry/overview/FormEditPage.tsx

```diff
@@ -8,7 +8,7 @@ import Loading from 'src/common/Loading'
 import TextInputDialog from 'src/common/TextInputDialog'
 import UnsavedChangesContext from 'src/contexts/unsavedChangesContext'
 import EntryCardHistoryDialog from 'src/entry/overview/EntryCardHistoryDialog'
-import EntryCardRolesDialog from 'src/entry/overview/EntryCardRolesDialog'
+import EntryCardRolesDialog from 'src/entry/overview/EntryRolesDialog'
```

Comment:
`EntryCardRolesDialog` should be renamed to match the new name of the component: `EntryRolesDialog`

Reply:
Import name still needs updating